commit 7e7c6498841cb4e5a5b4454424a374b70bb111d9
Author: Kai Koehne <kai.koehne@qt.io>
Date:   Fri Apr 3 10:39:14 2020 +0200

    CMake: Fix double inclusion of CMake plugin targets
    
    After 99ace38d22c640e37bb1a41095ee3b126169816a, all plugin files are
    included automatically, not only the ones ending in Plugin.cmake.
    
    Thus the extra inclusion done by the QmlConfigExtras file should only
    be done if strict mode is set.
    
    Amends 99ace38d22c640e37bb1a41095ee3b126169816a
    Amends 2f2dd3b0c28db210ea1f00d569f6c1626894c5f4
    
    Task-number: QTBUG-83282
    Change-Id: I416cbad6c4788d605a9a74f21062543c9c98e968
    Reviewed-by: Alexandru Croitor <alexandru.croitor@qt.io>

diff --git a/src/widgets/Qt5WidgetsConfigExtras.cmake.in b/src/widgets/Qt5WidgetsConfigExtras.cmake.in
index 236d1d4159..f05dae23ea 100644
--- a/src/widgets/Qt5WidgetsConfigExtras.cmake.in
+++ b/src/widgets/Qt5WidgetsConfigExtras.cmake.in
@@ -14,7 +14,8 @@ if (NOT TARGET Qt5::uic)
     )
 endif()
 
-include(\"${CMAKE_CURRENT_LIST_DIR}/Qt5Widgets_AccessibleFactory.cmake\" OPTIONAL)
+if (QT5_STRICT_PLUGIN_GLOB OR Qt5$${CMAKE_MODULE_NAME}_STRICT_PLUGIN_GLOB)
+    include(\"${CMAKE_CURRENT_LIST_DIR}/Qt5Widgets_AccessibleFactory.cmake\" OPTIONAL)
 
 set(Qt5Widgets_UIC_EXECUTABLE Qt5::uic)
 
